frame/nomination-pools/src/migration.rs

```diff
@@ -724,4 +724,99 @@ pub mod v5 {
 			Ok(())
 		}
 	}
+
+	/// This migration accumulates and initializes the [`TotalValueLocked`] for all pools.
+	pub struct VersionUncheckedMigrateV5ToV6<T>(sp_std::marker::PhantomData<T>);
```

Comment:
I hope nobody is us using nomination pools on parachains, but we should mention in the migration that it is using lots of weight and may not be suitable for parachains.

Reply:
Thanks for pointing it out. How do you think this mention should be done? Just a comment in the code?